apps/src/music/views/beatpad.module.scss

```diff
@@ -79,14 +79,21 @@ $hover-opacity: 0.5;
   flex: 1;
   margin-right: 10px;
   border-width: 3px;
+  cursor: pointer;
```

Comment:
Out of scope for this, but looks like this necessary becasue it's a clickable div? Can we use button elements?

Reply:
Oh good catch! I'll file a ticket to convert these to buttons